e2e_tests/tests/fixtures/estimator_dataset/model.py

```diff
@@ -31,14 +31,35 @@
 from the analytical calculations. Replace this model with a more robust one.
 
 """
+from typing import Any, List
+
 import numpy as np
 import tensorflow as tf
 
-from determined.estimator import EstimatorTrial, EstimatorTrialContext
+from determined import estimator
+
+
+def sum_reducer(batch_metrics: List):
+    """A function that is able to operate as a custom reducer."""
+    return np.hstack(batch_metrics).sum()
+
+
+class SumReducer(estimator.MetricReducer):
+    """A class that is able to operate as a custom reducer."""
+
+    def __init__(self):
+        self.sum = 0
 
+    def accumulate(self, metric: Any):
+        self.sum += metric.sum()
+        return self.sum
 
-class EstimatorDebugTrial(EstimatorTrial):
-    def __init__(self, context: EstimatorTrialContext):
+    def cross_slot_reduce(self, per_slot_metrics: List):
+        return sum(per_slot_metrics)
+
+
+class EstimatorDebugTrial(estimator.EstimatorTrial):
```

Comment:
non-blocking: can we rename this from Debug to something else

Reply:
sure, done